src/components/ui/inputs/TextInput.tsx

```diff
@@ -0,0 +1,46 @@
+'use client';
+import { useId } from 'react';
+import { clsx } from 'clsx';
+
+type TextInputProps = {
+  name?: string;
+  type?: string;
+  className?: string;
+  label: string;
+  id: string;
```

Comment:
Why are id field and label field required? What if we don't want to have a label? 
Also, if you want to haev the logic of label-input connection, you don't need to pass an id from the outside. You can handle the connection inside the component using the useId hook

Reply:
Yes, thank you